array-manipulation-3/src/union.js

```diff
@@ -0,0 +1,13 @@
+/* exported union */
+function union(first, second) {
+  var newArray = [];
+  for (var i = 0; i < first.length; i++) {
+    newArray.push(first[i]);
+  }
```

Comment:
Same feedback here! What if `first` includes multiple values that are the same? We only want one of each value in the new array to be returned.